tests/assert_news/test_news_file_validation.py

```diff
@@ -1,20 +1,17 @@
-"""Tests for mbed_tools_ci.assert_news.NewsFileValidator."""
 from unittest import mock, TestCase
 from pyfakefs.fake_filesystem_unittest import Patcher
 
 from mbed_tools_ci.assert_news import (
-    NewsFileValidator,
     NEWS_FILE_NAME_REGEX,
+    NewsFileValidator,
+    validate_news_file,
 )
 
 
 class TestValidateFileName(TestCase):
-    """Tests for the NewsFileValidator.validate_file_name method."""
-
     def test_raises_if_file_name_is_not_valid(self):
-        """Given an invalid file name, it raises ValueError."""
         with self.assertRaises(ValueError) as cm:
-            NewsFileValidator("imaginary/path/to/file.tar.gz").validate_file_name()
+            NewsFileValidator("/imaginary/path/to/file.tar.gz").validate_file_name()
```

Comment:
Just out of interest - why did you change this?

Reply:
It was to make the absoluteness of paths more explicit (in code and tests). They are not strictly necessary, provided you're in good directory and news files are available relatively to where you are.

There's some hairy stuff going on with relative vs absolute paths in this code anyway. `configuration.` returns absolute paths always (which is good), but `GitWrapper` returns changes relative to project root. I decided to use `absolute` paths always for clarity.

Reply:
Hmm yeah its something I haven't been very aware of so far. Will keep it in mind for any future stuff.